compiler/prelude/prelude.go

```diff
@@ -209,6 +209,11 @@ var $bytesToString = function(slice) {
   if (slice.$length === 0) {
     return "";
   }
+
+  if ($global.Buffer !== "undefined") {
+    return Buffer.from(slice.$array).toString("binary", slice.$offset, slice.$offset + slice.$length);
+  }
+
   var str = "";
   for (var i = 0; i < slice.$length; i += 10000) {
```

Comment:
@neelance @shurcooL - what's special about the value `10,000`?

I note the Google Closure compiler seems to have a rather specific `CHUNK_SIZE` value... no idea where they got the value from however (iterative solution based on some performance tests perhaps?):

https://github.com/google/closure-library/blob/e877b1eac410c0d842bcda118689759512e0e26f/closure/goog/crypt/crypt.js#L53-L71